apps/src/templates/teacherDashboard/SectionActionDropdown.jsx

```diff
@@ -0,0 +1,271 @@
+import React, {Component, PropTypes} from 'react';
+import color from "../../util/color";
+import {sortableSectionShape, OAuthSectionTypes} from "./shapes.jsx";
+import PopUpMenu from "@cdo/apps/lib/ui/PopUpMenu";
+import i18n from '@cdo/locale';
+import {pegasus} from "../../lib/util/urlHelpers";
+import {sectionCode,
+        sectionName,
+        removeSection,
+        toggleSectionHidden,
+        importOrUpdateRoster} from './teacherSectionsRedux';
```

Comment:
Nit: We usually avoid aligned indentation, would prefer
```js
import {
  sectionCode,
  //...
  importOrUpdateRoster,
} from './teacherSectionsRedux';
```
Bonus points if you can find the eslint rule that would enforce and/or autofix this :grin: